packages/eds-core-react/src/components/Icon/Icon.tsx

```diff
@@ -108,7 +108,7 @@ export const Icon = forwardRef<SVGSVGElement, IconProps>(function Icon(
   }
 
   const pathProps = {
```

Comment:
It might be good with a better way of setting pathProps? Perhaps rewriting it to be an array or something? 🤔 

Alternatively remove the `d` prop, and pass the pathData directly instead?

Reply:
Yes good point. I removed `d` from pathData and set it directly instead